marklogic-data-hub/src/main/java/com/marklogic/hub/impl/DataHubImpl.java

```diff
@@ -254,26 +254,26 @@ public boolean isServerVersionValid(String versionString) {
                 return false;
             }
             if(serverVersion.isNightly()){
-                //Support all 10.0-nightly on or after 6/20/2019 and 9.0-nightly on or after 7/25/2019
+                //Support all 10.0-nightly on or after 9/12/2019 and 9.0-nightly on or after 11/12/2019
                 if(serverVersion.getMajor() == 9) {
-                    Date minDate = new GregorianCalendar(2019, Calendar.JULY, 25).getTime();
+                    Date minDate = new GregorianCalendar(2019, Calendar.NOVEMBER, 12).getTime();
                     Date date = new SimpleDateFormat("y-M-d").parse(serverVersion.getDateString());
                     if (date.before(minDate)) {
                         return false;
                     }
                 }
                 if(serverVersion.getMajor() == 10) {
-                    Date minDate = new GregorianCalendar(2019, Calendar.JUNE, 20).getTime();
+                    Date minDate = new GregorianCalendar(2019, Calendar.SEPTEMBER, 12).getTime();
                     Date date = new SimpleDateFormat("y-M-d").parse(serverVersion.getDateString());
                     if (date.before(minDate)) {
                         return false;
                     }
                 }
             }
-            //5.1.0 supports server versions 9.x >= 9.0-10 and 10.x >= 10.0.2
+            //5.3.0 supports server versions 9.x >= 9.0-11 and 10.x >= 10.0.2
             else {
                 if(serverVersion.getMajor() == 9){
-                    if(serverVersion.getMinor() < 1000) {
+                    if(serverVersion.getMinor() < 1100) {
```

Comment:
Same thing - why do we require 9.0-11 and not 9.0-10?

Reply:
@kkanthet mentioned in the ticket that the support for 5.3.0 starts from 9.0-11. I am not sure of the specific reason why 9.0-10 is not supported.

Reply:
Please try to find the reason for it - this is likely the best place to have some documentation, since it's the code that's enforcing the requirement.